content/installation/upgrade.md

```diff
@@ -11,6 +11,16 @@ Custom installations do not support client-side upgrade functionality.
 Use [`kubectl directpv migrate`]({{< relref "/command-line/migrate.md" >}}) to migrate the old resources to a new installation.
 {{< /admonition >}}
 
+{{< admonition title="Pod Security Policies" type="note" >}}
+The `PodSecurityPolicy` feature was deprecated in Kubernetes v1.21 and removed in v1.25.
+The oldest supported release of Kubernetes is v1.28.
```

Comment:
1.28 is eol at the end of October. I guess we could link to the Kubernetes releases page instead?

Reply:
Removed, as I decided it just wasn't important to talk about supported versions of K8s in this admonition.